build.sbt

```diff
@@ -2,28 +2,27 @@ import Defaults._
 
 lazy val root = (project in file("."))
   .settings(commonSettings: _*)
-  .enablePlugins(CrossPerProjectPlugin)
   .aggregate(NGHelpers, NGPlugin)
 
 lazy val NGHelpers = (project in file("helpers"))
   .settings(commonSettings: _*)
   .settings(
     name := "play-testng-helpers",
-    scalaVersion := "2.11.11",
+    scalaVersion := "2.12.4",
     crossScalaVersions := Seq("2.11.11", "2.12.4"),
     libraryDependencies ++= Seq(
-      "org.testng" % "testng" % "6.8.8", // % "provided"
-      "com.typesafe.play" %% "play-test" % "2.5.18", //% "provided"
-      "com.typesafe.play" %% "play-java" % "2.5.18" //% "provided"
+      "org.testng" % "testng" % "6.11", // % "provided"
+      "com.typesafe.play" %% "play-test" % "2.6.7", //% "provided"
+      "com.typesafe.play" %% "play-java" % "2.6.7" //% "provided"
     ))
 
 lazy val NGPlugin = (project in file("plugin"))
   .settings(commonSettings: _*)
   .settings(
     name := "play-plugins-testng",
     sbtPlugin := true,
-    scalaVersion := "2.10.6",
-    crossScalaVersions := Seq("2.10.6", "2.11.11", "2.12.4"),
```

Comment:
These need to remain 2.10. We're not ready to bump Scala or SBT version for sbt plugin libraries.